java/src/jmri/server/json/package-info.java

```diff
@@ -139,9 +139,10 @@
  * Starting with 5.0.0, the JSON protocol version follows semantic version
  * rules, prior to that the version is just a major.minor version.
  * <dl>
- * <dt>5.1.1 (JMRI 4.17.7)</dt>
+ * <dt>5.2.0 (JMRI 4.17.7)</dt>
```

Comment:
This needs to match the JSON_PROTOCOL_VERSION in the class `jmri.server.json.JSON`